core/block_render_svg_vertical.js

```diff
@@ -730,6 +747,31 @@ Blockly.BlockSvg.prototype.renderFields_ =
   return this.RTL ? -cursorX : cursorX;
 }; /* eslint-enable indent */
 
+/**
+ * Add a vertical separator line SVG after the current field.
+ * @param {Blockly.Field} field - the field after which to place the line.
+ * @param {number} cursorX - the current x position for rendering.
+ * @param {number} cursorY - the current y position for rendering.
+ * @private
+ */
+Blockly.BlockSvg.prototype.addIconSeparatorLine_ = function(field, cursorX,
+    cursorY) {
+  var lineTopHeight = Blockly.BlockSvg.ICON_SEPARATOR_HEIGHT / 2;
+  var lineBottomHeight = lineTopHeight;
+  // If this is a hat block, shorten the bottom of the line by one grid unit.
+  if (!this.previousConnection && this.nextConnection) {
+    lineBottomHeight -= Blockly.BlockSvg.GRID_UNIT;
+  }
+  Blockly.utils.createSvgElement('line', {
```

Comment:
This creates a new element every time it's called, and appends it to the source block's svg group.  You'll need to keep a handle on it to avoid creating duplicate lines on every render.